modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/logit/storage/file/AbstractFile.java

```diff
@@ -120,6 +121,7 @@ public void map(final MapMode mapMode) {
                 try (final RandomAccessFile randomAccessFile = new RandomAccessFile(this.file, "rw");
                         final FileChannel fileChannel = randomAccessFile.getChannel()) {
                     this.mappedByteBuffer = fileChannel.map(mapMode, 0, this.fileSize);
+                    mappedByteBuffer.order(ByteOrder.LITTLE_ENDIAN);
```

Comment:
How about introducing a constant somewhere for the byte order we use in the logic storage? Otherwise, these `ByteOrder.LITTLE_ENDIAN` are scattered here and there and it will be a bit more difficult to track them all down (if we need to)

Reply:
Sure